config/system/alt.yaml

```diff
@@ -30,3 +30,12 @@ run_trigger:
   vlan_start: 1001
   vlan_end: 1009
   egress_vlan: 121
+
+mininet_subnet:
+  base_ip: 192.168.1.0
+  prefix_length: 24
+
+# Define external DHCP subnets to be used by test hosts
+external_dhcp_subnets:
+  - base_ip: 10.20.99.0
```

Comment:
I would go with the CIDR nomenclature... it's much more common and concise:

10.20.99.0/24